packages/matchbox/src/components/ThemeProvider/ThemeProvider.js

```diff
@@ -18,22 +39,25 @@ const GlobalStyle = createGlobalStyle`
  *
  * @prop theme Overrides matchbox's theme
  */
-
 function Theme(props) {
   return (
-    <ThemeProvider theme={{ ...theme, ...props.theme }}>
-      {/* 
-        FE-913:`skipGlobalStyles` is to exclude global styles from unit tests.
-        The custom assertion `toHaveAttributeValue` fails with this turned on.
-        We should either move away from using this assertion (or even enzyme),
-        or try to resolve the error
-      */}
-      {!props.skipGlobalStyles && <GlobalStyle />}
-      {props.children}
-    </ThemeProvider>
+    <Manager>
+      <ThemeProvider theme={{ ...theme, ...props.theme }}>
+        {/*
+          FE-913:`skipGlobalStyles` is to exclude global styles from unit tests.
+          The custom assertion `toHaveAttributeValue` fails with this turned on.
+          We should either move away from using this assertion (or even enzyme),
+          or try to resolve the error
+        */}
+        {!props.skipGlobalStyles && <GlobalStyle />}
+
+        {props.children}
+      </ThemeProvider>
+    </Manager>
   );
 }
 
+Theme.displayName = 'MatchboxThemeProvider';
 Theme.propTypes = {
```

Comment:
Should probably include:
```
target: PropTypes.element
```

Reply:
`element` includes only react elements :/
Added `object` here, although it isnt very helpful